Day1/P1.py

```diff
@@ -0,0 +1,300 @@
+            txt = txt[0:i] + 'N' + txt[i+1:]
+        elif txt[i] == 'o':
+            txt = txt[0:i] + 'O' + txt[i+1:]
+        elif txt[i] == 'p':
+            txt = txt[0:i] + 'P' + txt[i+1:]
+        elif txt[i] == 'q':
+            txt = txt[0:i] + 'Q' + txt[i+1:]
+        elif txt[i] == 'r':
+            txt = txt[0:i] + 'R' + txt[i+1:]
+        elif txt[i] == 's':
+            txt = txt[0:i] + 'S' + txt[i+1:]
+        elif txt[i] == 't':
+            txt = txt[0:i] + 'T' + txt[i+1:]
+        elif txt[i] == 'u':
+            txt = txt[0:i] + 'U' + txt[i+1:]
+        elif txt[i] == 'v':
+            txt = txt[0:i] + 'V' + txt[i+1:]
+        elif txt[i] == 'w':
+            txt = txt[0:i] + 'W' + txt[i+1:]
+        elif txt[i] == 'x':
+            txt = txt[0:i] + 'X' + txt[i+1:]
+        elif txt[i] == 'y':
+            txt = txt[0:i] + 'Y' + txt[i+1:]
+        elif txt[i] == 'z':
+            txt = txt[0:i] + 'Z' + txt[i+1:]
+    return txt
+
+print(upperstr('Hello, And Welcome To My World!'))
+print(upperstr('HSKHNKXjkjskfeKWEOIIHF'))
+print(upperstr('HELLO HOW ARE YOU'))
+print('-'*20)
+
+#count the number of times a specified value appears in the string
+def countvalue(txt,char,start=None,end=None):
+    count = 0
+    for i in range(len(findchars(txt,char,start,end))):
+        count += 1
+    return count
+
+print(countvalue('hello my name is tram','a'))
+print(countvalue('hello my name is tram','my'))
+print(countvalue('hello my name is tram','z'))
+print(countvalue('hello my name is tram','lo'))
+print(countvalue('hello my name is tram','a',12))
+print(countvalue('hello my name is tram','m',8,12))
+print(countvalue('ohmymymyabcdefmy','my'))
+print(countvalue('I love apples, apple are my favorite fruit','apple'))
+print('-'*20)
+
+#endswith() - returns true if the string ends with the specified value
+#def isendswith(txt,char)
+
+#isupper() - returns True if all characters in the string are upper case
+def isuppercase(txt):
+    if txt == upperstr(txt):
+        return True
+    return False
+
+print(isuppercase('Hello World!'))
+print(isuppercase('hello 123'))
+print(isuppercase('MY NAME IS PETER'))
+print('-'*20)
+
+#islower() - returns True if all characters in the string are lower case
+def isuppercase(txt):
+    if txt == lowerstr(txt):
+        return True
+    return False
+
+print(isuppercase('Hello World!'))
+print(isuppercase('hello 123'))
+print(isuppercase('mynameisPeter'))
+print('-'*20)
+
+#isalpha() - returns True if all characters in the string are in the alphabet
+def isalphabet(txt):
+    for i in range(len(txt)):
+        if txt[i] == 'A' or 'a' or 'B' or 'b' or 'C' or 'c' or 'D' or 'd' or 'E' or 'e' or 'F' or 'f' or 'G' or 'g' or 'H' or 'h' or 'I' or 'i' or 'J' or 'j' or 'K' or 'k' or 'L' or 'l' or 'M' or 'm' or 'N' or 'n' or 'O' or 'o' or 'P' or 'p' or 'Q' or 'q' or 'R' or 'r' or 'S' or 's' or 'T' or 't' or 'U' or 'u' or 'V' or 'v' or 'W' or 'w' or 'X' or 'x' or 'Y' or 'y' or 'Z' or 'z':
```

Comment:
Ask yourself, what is the task you are doing here??? Can you group the characters into something so that you can check easier???